Xamarin.Forms.Xaml.UnitTests/Issues/Bz54334.xaml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms" xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" x:Class="Xamarin.Forms.Xaml.UnitTests.Bz54334">
+<StackLayout Padding="10">
+        <Label x:Name="label" HorizontalTextAlignment="Center" Text="I have a set textcolor, but in since pre-3 styles will override it.."  TextColor="Black" />
+        <Label x:Name="themedLabel" HorizontalTextAlignment="Center" Text="I dont have a set textcolor, syles can override me as much as they want"   />
```

Comment:
small thing: typo. `syles` instead of `styles`. also missing an apostrophe, but whatever.